src/core/public/chrome/ui/header/header.tsx

```diff
@@ -281,93 +284,186 @@ export function Header({
     );
   };
 
-  const renderLeftControls = () => (
-    <>
-      {useUpdatedHeader && (
-        <EuiHeaderSectionItem border="none">
-          <HeaderControlsContainer
-            data-test-subj="headerLeftControl"
-            controls$={application.currentLeftControls$}
-          />
-        </EuiHeaderSectionItem>
-      )}
+  const renderLeftControls = () => {
+    let hasLeftControls;
+    let hasNavControlsLeft;
 
-      {/* Nav controls left */}
-      <EuiHeaderSectionItem border={useUpdatedHeader ? 'none' : 'right'}>
-        <HeaderNavControls side="left" navControls$={observables.navControlsLeft$} />
-      </EuiHeaderSectionItem>
-    </>
-  );
+    application.currentLeftControls$.subscribe((value) => {
+      hasLeftControls = isTruthy(value);
+    });
+    observables.navControlsLeft$.subscribe((value) => {
+      hasNavControlsLeft = isTruthy(value);
+    });
 
-  const renderCenterControls = () => (
-    <>
-      {useUpdatedHeader && (
-        <EuiHeaderSectionItem border="none">
-          <HeaderNavControls navControls$={observables.navControlsExpandedCenter$} />
-        </EuiHeaderSectionItem>
-      )}
+    if (!hasLeftControls && !hasNavControlsLeft && useUpdatedHeader) {
+      return null;
```

Comment:
Are there any consumers that rely on the presence of the dom element to insert content? Mostly just asking if this sparks any known use cases we could potentially break here, but I'm hoping no one is doing manual dom lookups to insert content.

Reply:
No as far as I know, finger crossed